Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Controllers/NoticestoMarinersControllerTest.cs

```diff
@@ -103,6 +111,18 @@ public async Task WhenIndexPostIsCalledWithWeekOrYearZero_ThenShouldReturnsExpec
             Assert.AreEqual(week, Convert.ToInt32(((ViewResult)result).ViewData["Week"]));
         }
 
+        [Test]
```

Comment:
This name confused me. I think a better name would be WhenIndexPostIsCalledAndExceptionThrownByService_ThenShouldThrowException

Reply:
Changes done.